client/src/components/DashBoard/DashBoardContent.tsx

```diff
@@ -0,0 +1,59 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Box, AppBar, Toolbar, IconButton, Typography, Button } from '@mui/material';
+import { CalendarToday, Dashboard, Menu as MenuIcon, Add } from '@material-ui/icons';
+import logo from '../../Images/logo.png';
+import userLogo from '../../Images/d9fc84a0d1d545d77e78aaad39c20c11d3355074.png';
+
+export default function DashBoardContent(): JSX.Element {
+  return (
+    <div>
+      <div>
```

Comment:
I don't think you need this wrapping `div` here. The inner `Box` component already serves for that.